docs/quickstart/tardis_example.yml

```diff
@@ -0,0 +1 @@
+/home/parikshit/Desktop/tardis/docs/tardis_example.yml
```

Comment:
I guess you should use relative imports here. In **this** case, because this is a symlink for `docs/tardis_example.yml`, just `../tardis_example.yml` should work.

Reply:
thanks for rectifying the mistakes I really don't know how I missed them. Thanks again